src/System.Security.Cryptography.Algorithms/src/System/Security/Cryptography/SHA256.cs

```diff
@@ -36,21 +30,22 @@ public Implementation()
                 HashSizeValue = _hashProvider.HashSizeInBytes * 8;
             }
 
-            protected sealed override void HashCore(byte[] array, int ibStart, int cbSize)
-            {
+            protected sealed override void HashCore(byte[] array, int ibStart, int cbSize) =>
                 _hashProvider.AppendHashData(array, ibStart, cbSize);
-            }
 
-            protected sealed override byte[] HashFinal()
-            {
-                return _hashProvider.FinalizeHashAndReset();
-            }
+            protected override void HashCore(ReadOnlySpan<byte> source) =>
```

Comment:
> I don't want to do that work for this PR

It has nothing to do with not wanting to do work.  It has everything to do with disagreeing with the style.

> You will not use explicit 'sealed' anywhere

I have no intention of removing them everywhere; that's a larger discussion that would require more than just my opinion on the matter.  Due to the request for consistency, I removed them from the existing methods on the classes to which more methods were being added, so that we're not in a weird state where some methods on these types are explicitly sealed and others aren't.